core/tauri/src/updater/core.rs

```diff
@@ -622,12 +626,12 @@ impl<R: Runtime> Update<R> {
     // set our headers
     let mut headers = self.headers.clone();
     headers.insert(
-      "Accept",
-      HeaderValue::from_str("application/octet-stream").unwrap(),
+      header::ACCEPT,
+      HeaderValue::from_str("application/octet-stream")?,
     );
     headers.insert(
-      "User-Agent",
-      HeaderValue::from_str("tauri/updater").unwrap(),
+      header::USER_AGENT,
+      HeaderValue::from_str(UPDATER_USER_AGENT)?,
```

Comment:
we shouldn't change the user agent - just in case it breaks someone's server

Reply:
cool, reverted in [`035093e` (#10939)](https://github.com/tauri-apps/tauri/pull/10939/commits/035093ea14058c7c437f2b626b407900297411f8)